content/events/anvil2023-january-office-hours.md

```diff
@@ -0,0 +1,30 @@
+---
+author: "AnVIL"
+conference: "AnVIL 2023"
+description: "Helping users find solutions to their AnVIL questions."
+eventType: "Office Hours"
+featured: true
+location: "Virtual"
+sessions: [{sessionStart: "26 Jan 2023 11:00 AM"}]
+timezone: "America/New_York"
+title: "AnVIL Office Hours - February 2023"
```

Comment:
Hi @jpaten would you mind changing the month in the title from `February` to `January`?